src/content/docs/Products/SplashKit/Splashkit Expansion/Documentation/onboarding-guide.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: SplashKit Expansion Onboarding Guide
+sidebar:
+  label: SplashKit Expansion Onboarding Guide
+  order: 1
+---
+
+## Introduction
+
+This guide will cover all the steps required to get contributing to the splashkit-core repository.
+Feel free to skip steps which you have already completed or are familiar with.
+
+## Installing WSL
+
+WSL is a built-in Linux distribution virtual machine for Windows. splashkit-core will be installed
+to the Linux distribution. The official Microsoft documentation provides high-quality instructions
+on how to install WSL: [Install WSL](https://learn.microsoft.com/en-us/windows/wsl/install). This
+guide has been tested with the default Ubuntu distribution, but others may also work.
+
+### Installing Visual Studio Code
+
+Once WSL has been installed to Windows, Visual Studio Code needs to be installed to WSL.
+The Microsoft documentation is again the best reference for this process:
+[Install VS Code](https://learn.microsoft.com/en-us/windows/wsl/tutorials/wsl-vscode).
+
+### Installing Git
+
+Now Git must be installed to WSL. Follow the Microsoft installation instructions:
+[Install Git](https://learn.microsoft.com/en-us/windows/wsl/tutorials/wsl-git).
+
+### Installing Windows Terminal (optional)
+
+Windows Terminal is an updated Command Prompt with many useful features. It is not mandatory to
+install, however it is recommended due to its ease of use. More information about Windows Terminal
+can be found here: [Windows Terminal](https://learn.microsoft.com/en-us/windows/terminal/).It can be
+installed through the Microsoft Store.
+
+By default, new tabs will open as a Command Prompt, with WSL terminals being accessible with the
+down-arrow. Since WSL will be used so frequently, there is the option to change the default tab to
+WSL. Open the settings by clicking the down arrow and selecting ‘Settings’. In the ‘Startup’ tab,
+‘Default profile’ allows you to change the default tab type to WSL.
+
+## Setting Up splashkit-core
+
+Now that WSL is fully configured, it is time to install the splashkit-core repository.
+
+### Forking splashkit-core Repository
```

Comment:
Here you could possibly link to this as a general guide for forking - https://thoth-tech.netlify.app/products/splashkit/splashkit-website/onboarding/01-setting-up/